pom.xml

```diff
@@ -20,13 +20,13 @@
     <groupId>org.jboss.aerogear</groupId>
     <artifactId>unifiedpush-java-client</artifactId>
     <packaging>jar</packaging>
-    <version>1.1.0-alpha.1-SNAPSHOT</version>
+    <version>1.1.0-alpha.2-SNAPSHOT</version>
```

Comment:
My bad, I forgot to push the commit probably and cleaned my local repos since then :/